vm-setup/README.md

```diff
@@ -0,0 +1,28 @@
+# Debezium Microservices Lab Vagrant image
+
+This is a source repository for [Vagrant image](https://app.vagrantup.com/debezium/boxes/microservices-lab) serving as a Debezium microservices lab environment.
+
+## Image usage
+There are two ways how to use the image
+* (recommended) Use provided `Vagrant` file. In this case just download the `Vagrantfile` into an arbitrary directory and issue `vagrant up` command.
+After a while the new virtual machine will be spawned and you could log to it via `vagrant ssh`.
+* (advanced) Create the environment from scratch using `vagrant init debezium/microservices-lab` command.
+In this case the user is responsible for configuration of networking, port-forwarding and capacity allocation.
+
+## Building the image
+To build the image from scratch it is necessary to follow this procedure
+* Use `Vagrantfile.build`as the Vagrant configuration file.
```

Comment:
How do I do this exactly?

Reply:
Unfortunately only copy /rename - I've not found any other way